aspnetcore/migration/21-to-22.md

```diff
@@ -29,6 +29,16 @@ Projects targeting .NET Framework may continue to use the TFM of a version great
 <TargetFramework>net461</TargetFramework>
 ```
 
+## IIS in-process hosting model
```

Comment:
All other headings in the migration doc start with an action verb. I suggest doing the same here.

 ```suggestion
## Adopt IIS in-process hosting model
```